lamindb/models/schema.py

```diff
@@ -213,7 +213,9 @@ class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
     For a composite schema, the hash of hashes.
     """
     minimal_set: bool = BooleanField(default=True, db_index=True, editable=False)
-    """Whether the schema contains a minimal set of linked features (default `True`).
+    """Deprecated. Use `optional` instead.
```

Comment:
Do we still want to deprecate it? Isn't it useful to be able to set `minimal_set = False` in some cases and thereby make all features optional? 🤔

Reply:
OK!